skrifa/src/metrics.rs

```diff
@@ -370,6 +378,17 @@ impl GlyphMetrics<'_> {
         deltas[1] -= deltas[0];
         Some([deltas[0], deltas[1]].map(|delta| delta.x.to_i32()))
     }
+
+    fn bounds_from_outline(&self, glyph_id: GlyphId) -> Option<BoundingBox> {
+        if let Some(outline) = self.font.outline_glyphs().get(glyph_id) {
+            let settings = DrawSettings::unhinted(Size::unscaled(), self.coords);
```

Comment:
I guess we’ll need to save the size from the constructor as well so we can return an appropriately scaled bbox here.